HackYourFuture-CPH/rate-my-cv
# Endpoint to retrieve the last uploaded CVs #95

# Title :  Endpoint to retrieve the last uploaded CVs #95

One can  retrieve from the DB a list the 3 last upload CVs or All the whole list (limit to 20), so that you can view them on Frontend. 

Fixes # https://github.com/HackYourFuture-CPH/rate-my-cv/issues/95

# How to test?
On frontend, you can see the 3 last uploaded CVs as below; 
<img width="260" alt="last3uploadedCvs" src="https://user-images.githubusercontent.com/58881390/107674462-48130680-6c97-11eb-9985-9f2361a9dc6c.PNG">

On backend Testing,., I joined users and cv tables, I created new usercv router and usercv controller. I also limit the cvs numbers to be fetched from backend to max 20 cvs. see joined table below; 
<img width="528" alt="users+cvs_JOINED_tables" src="https://user-images.githubusercontent.com/58881390/107674710-86a8c100-6c97-11eb-9d9f-348baea5374d.PNG">




# Checklist

- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] This PR is ready to be merged and not breaking any other functionality
